docs/usage.rst

```diff
@@ -90,19 +88,36 @@ regular TestCases.  So you can do something like:
 
 And you don’t have to clean the database yourself.
 
-If you have `factories`_ you want to instantiate on a per-scenario basis,
-you can initialize them in ``environment.py`` like this:
+django_ready hook
+-----------------
+
+You can add a ``django_ready`` method in your ``environment.py`` file in case
```

Comment:
Ah, it's actually a _function_, not a method. But nevermind. No need to be that picky.